src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -49,33 +50,29 @@ def test_no_space_cont_create(self):
         err2 = "-1009"
 
         # create pool
-        self.pool = DaosPool(self.context)
-        mode = self.params.get("mode", '/conttests/createmode/')
-        self.d_log.debug("mode is {0}".format(mode))
-        uid = os.geteuid()
-        gid = os.getegid()
-        # 16 mb pool, minimum size currently possible
-        size = 16777216
+        self.pool = TestPool(self.context, dmg_command=self.get_dmg_command())
+        self.pool.get_params(self)
 
         self.d_log.debug("creating pool")
-        self.pool.create(mode, uid, gid, size, self.server_group, None)
+        self.pool.name.value = self.server_group
```

Comment:
This could be handled by placing "daos_server" as the `/run/pool/name` entry in the test yaml file - which is actually the default, so it could be removed all together.